### PR TITLE
Table render parameters

### PR DESCRIPTION
**Pull request by jeffhsta**
We have improved the script the get the information from swagger and render it on our documentation.

We now have a new tag `[[prefix:operation:operation-name:parameters]]` like `[[telemetry:operation:getBrands:parameters]]` it will be replaced by a markdown table.

pairing with @lucaslago 

The tag `[[telemetry:operation:getBrands:parameters]]` would look like this:

screenshot from 2016-08-25 18-04-39

### CODE REVIEW

File: lib/template_handler.rb [hunk 1]
```diff
@@ -3,14 +3,21 @@
 # The TemplateHandler class is responsible for applying a given
 # SwaggerDefinition to the markdown documentation file.
 class TemplateHandler
-  def initialize(template)
+  def initialize(template, table_template)
```
Comment (lucaslago) [inline lib/template_handler.rb]: s/template/json_template

File: spec/template_handler_spec.rb [hunk 2]
```diff
@@ -8,12 +8,16 @@
...
+    double 'table_template'
+  end
+
   let :file do
     double 'file'
   end
 
   subject do
-    TemplateHandler.new(template)
+    TemplateHandler.new(template, table_template)
```
Comment (lucaslago) [inline spec/template_handler_spec.rb]: s/template/json_template

**Comment by waldemarnt** [comment]
@jeffhsta can you paste some screenshot?

**Comment by jeffhsta** [comment]
@waldemarnt we didn't change the documentation it self, just create that feature that we will use in another story, but I can take a screenshot of one example to be clear how it gonna look like.

**Comment by waldemarnt** [comment]
🚢 🇮🇹

**Pull request merged**